torchvision/transforms/v2/_misc.py

```diff
@@ -247,6 +254,34 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class SanitizeBoundingBoxes(Transform):
+    """[BETA] Remove degenerate/invalid bounding boxes and their corresponding labels.
```

Comment:
```suggestion
    """[BETA] Remove degenerate/invalid bounding boxes and their corresponding labels and optionally masks.
```

This also needs to go into the text below.